src/event_loop.rs

```diff
@@ -377,6 +309,30 @@ impl<T> EventLoopWindowTarget<T> {
     pub fn listen_device_events(&self, allowed: DeviceEvents) {
         self.p.listen_device_events(allowed);
     }
+
+    /// Sets the [`ControlFlow`].
+    pub fn set_control_flow(&self, control_flow: ControlFlow) {
+        self.p.set_control_flow(control_flow)
+    }
+
+    /// Gets the current [`ControlFlow`].
+    pub fn control_flow(&self) -> ControlFlow {
+        self.p.control_flow()
+    }
+
+    /// Send a [`LoopExiting`] event and stop the event loop.
+    ///
+    /// [`LoopExiting`]: Event::LoopExiting
+    pub fn exit(&self) {
```

Comment:
The issue here is that we don't actually send anything, we basically say that we want to exit event loop.

Reply:
So `terminate` and `is_terminating` are better names?

Reply:
I assume the problem here was the documentation, which I adjusted.

Reply:
We have an event called `LoopExiting`, not `LoopTerminating`, so using the word terminate will be inconsistent with the event names.